e2e/cypress/support/index.js

```diff
@@ -17,6 +17,8 @@ import 'cypress-file-upload';
 import 'cypress-wait-until';
 import addContext from 'mochawesome/addContext';
 
+require('cypress-plugin-tab');
```

Comment:
Please use import for consistency.  Also, please rearrange import order.
```suggestion
import 'cypress-plugin-tab';
```